runtime/bcutil/cfreader.c

```diff
@@ -3140,19 +3140,10 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
 	/* Currently value type is built on JDK22, so compare with JDK22 for now. */
```

Comment:
This comment should have been updated with the change on line 3143.

Reply:
Thanks Keith. I have another pull request in the works to close out https://github.com/eclipse-openj9/openj9/issues/18829 I will update the comment there.